core/src/main/java/org/apache/iceberg/inmemory/InMemoryCatalog.java

```diff
@@ -315,7 +315,7 @@ public void doCommit(TableMetadata base, TableMetadata metadata) {
 
       if (null == base && !namespaceExists(tableIdentifier.namespace())) {
         throw new NoSuchNamespaceException(
-            "Cannot create table %s. Namespace %s does not exist",
+            "Cannot create table %s. Namespace does not exist: %s",
```

Comment:
I guess the original message was written in that way to avoid 2 colons. I am fine either way, let me know what you think.

Reply:
I think having `...Namespace does not exist: %s` is preferrable and we're already using it throughout that file (we just had a few minor inconsistencies in error reporting)

Reply:
In that case, could you help make all of them consistent by something like "Cannot list tables for namespace. Namespace does not exist: %s"? Because some places use `:` and some places use `.` for the first sentence at this moment even after your change.

Reply:
good point, I've updated those error messages to use `.` after the first sentence